_data/sites.json

```diff
@@ -7316,6 +7316,16 @@
         ]
     },
 
+    {
+        "name": "Followlike",
+        "url": "https://www.followlike.net/settings.php",
+        "difficulty": "easy",
+        "notes": "On Delete Sites or Account section, click the red 'Delete account'.",
```

Comment:
```suggestion
        "notes": "On Delete Sites of Account section, click the red 'Delete account'.",
```
Should this 'or' have been an 'of'? Sentence reads a bit strangely

Reply:
it actually `Delete Sites or Account`

![image](https://github.com/jdm-contrib/jdm/assets/90754622/c96525d1-cae5-49e0-99dc-a0225d03c7a3)

Reply:
Ahhh, I see, thanks !